LDAP-primer.md

```diff
@@ -291,6 +307,9 @@ and supported by all APIs, command line tools and applications. It's a bit quirk
 the parentheses, especially if multiple terms and negations are involved, but not too difficult
 to understand and write for most day-to-day uses.
 
+You need to be as careful writing LDAP filters that contain data from a user, as you do
+writing SQL statements, to avoid security problems.
+
```

Comment:
While true, this is a bit vague, and again, information overload, so I'll leave it out.